packages/plugin-github-pr-link/README.md

```diff
@@ -0,0 +1,44 @@
+# GitHub Pull Request link
+
+This plugin allows showing the Preevy environment URLs in a GitHub PR comment. The comment is added/updated at the `up` command and deleted in the `down` command.
+
+![Demo comment](./demo.png)
+
+## Configuration
+
+Add the plugin to the `plugins` section of the `x-preevy` element in your Docker Compose file:
+
+```yaml
```

Comment:
Not sure it's easy since the only thing currently configuring a plugin is the compose file.

We can add another compose file which has the plugin definition, but that would mean the "compose files" argument would become required, which is awkward.

I guess we need another way to indicate that a plugin is enabled. Maybe env var?

Reply:
Env var can work as well.
Basically, we already have mechanism for disabling the plugin, could be similar

Reply:
I wan to avoid additional configuration file and flag at this point, why it's needed? 

Also, it doesn't solve the GH action issue, it makes it more complex compared to just setting env bar.

Reply:
Got it, it support multiple flags and merging like compose

Reply:
It can work well, and we'll need such file at some point, the only thing I'm a bit worried, is what if we'll want/need a different configuration format.
Enabling/disabling plugin with env var seems relatively simple.

Reply:
Env vars are a limited solution If you need to specify a configuration for the plugin.